### DISCUSSION TITLE
Download the PDF in dark mode

### DISCUSSION THREAD

**Discussion by rafisics** [fjwillemsen/NativeOverleaf#18] [Q&A]
I am enjoying working in the dark mode with NativeOverleaf. I like to download the dark version of the PDF as well. How can I do it?

**Comment by fjwillemsen** [comment]
Hi Rafisics, good to hear that you like it! I'll look into whether this is possible, but I'm a bit hestitant to do this, because most people will expect their PDF to be untouched.

**Reply by rafisics** [reply]
>  most people will expect their PDF to be untouched

I agree. But nowadays, instead of printing out PDFs, many people including me prefer reading on screen and also in dark mode. So I really feel the necessity of getting the amazing dark-mode PDF that NativeOverleaf shows.
Also, I like to know if it's possible to provide the LaTeX preamble code that can reproduce PDF with the same dark-mode.

**Reply by fjwillemsen** [reply]
Yes, absolutely! But the problem is, I do not change the LaTeX or the PDF file itself in my implementation - I apply a filter to Overleaf's built-in PDF viewer. If someone finds a way to change the PDF itself on the fly using such a filter I would happily implement it, but for now I would recommend using a PDF viewer that does dark mode conversion.

**Reply by rafisics** [reply]
> but for now I would recommend using a PDF viewer that does dark mode conversion.

Your filter is comparatively better than the dark mode conversion of the PDF viewers I use.

**Reply by fjwillemsen** [reply]
That is great to hear, and I would love to implement it. I just have no idea how to apply a filter on the PDF file itself, so I will create an issue and request community help.
